packages/@aws-cdk/aws-codepipeline-actions/test/cloudformation/integ.stacksets.ts

```diff
@@ -0,0 +1,60 @@
+import * as codepipeline from '@aws-cdk/aws-codepipeline';
+import { Asset } from '@aws-cdk/aws-s3-assets';
+import { App, Stack, StackProps } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import * as cpactions from '../../lib';
+
+export class StackSetPipelineStack extends Stack {
+  constructor(scope: Construct, id: string, props: StackProps) {
+    super(scope, id, props);
+
+    const pipeline = new codepipeline.Pipeline(this, 'Pipeline');
+
+    const asset = new Asset(this, 'Asset', {
+      path: `${__dirname}/test-artifact`,
+    });
+
+    const sourceOutput = new codepipeline.Artifact('SourceArtifact');
+
+    pipeline.addStage({
+      stageName: 'Source',
+      actions: [
+        new cpactions.S3SourceAction({
+          actionName: 'Source',
+          output: sourceOutput,
+          bucket: asset.bucket,
+          bucketKey: asset.s3ObjectKey,
+        }),
+      ],
+    });
```

Comment:
Can we actually use a separate S3 Bucket with `RemovalPolicy.DESTROY`? And use it for both the Pipeline, and the source, like [this](https://github.com/aws/aws-cdk/blob/ccc198864f92620857da09c68013123e9cd3f01d/packages/%40aws-cdk/aws-codepipeline-actions/test/integ.pipeline-cfn-cross-region.ts#L15-L29)?

Otherwise, running integ tests leaves garbage in your account, which I don't like.